### PR TITLE
Properly format post_date for display

### PR DESCRIPTION
**Pull request by jamesgol**
This PR is incomplete but adds a branch with a test that we can use when we finally can fix it properly.

### CODE REVIEW

**Comment by jamesgol** [comment]
Related to #4525

File: tests/phpunit/includes/MagicTags/Test_Date.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		$id = $pod->add( array( 'testtext' => 'RIP Fats Domino', 'post_date' => '2017-10-25 12:34' ) );
+		$this->assertGreaterThan( 0, $id );
+
+		$pod->fetch( $id );
+		$result = $pod->do_magic_tags( '{@post_date}' );
+		$this->assertEquals( 'October 25, 2017', $result );
+
+		// Some additional tests which aren't exactly related to the Magic tag but further test the Pods functionality
+		// Eventually these should get broken off to some field/post date specific tests
+		$this->assertEquals( 'October 25, 2017', get_the_date( null, $id ) );
```
Comment (GaryJones) [inline tests/phpunit/includes/MagicTags/Test_Date.php]: This assertion is dependent on the Settings -> Date Format being a specific value. If WP changes the default value, or another unit tests updates the value and doesn't change it back, this assertion will fail.

File: tests/phpunit/includes/MagicTags/Test_Date.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * @package Pods_Unit_Tests
+ * @group pods_magictags
+ * @group pods_acceptance_tests
+ */
+class Date extends \Pods_Unit_Tests\Pods_UnitTestCase {
+	/**
+	 * @group pods-issue-4525
+	 */
+	public function test_create_template() {
+		$pod_name = __FUNCTION__;
```
Comment (sc0ttkclark) [inline tests/phpunit/includes/MagicTags/Test_Date.php:14]:
```suggestion
		update_option( 'date_format', 'F j, Y' );

		$pod_name = __FUNCTION__;
```

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
This is also related to #5676
That PR fixes the issue for #4525 and adds a feature to modify the format. The notation for this needs your review & might be best to discuss before merging..

Perhaps we can merge this PR into #5676?

**Comment by sc0ttkclark** [comment]
Closing in favor of https://github.com/pods-framework/pods/pull/5676

**Pull request closed without merging**